torch/csrc/jit/tensorexpr/kernel.cpp

```diff
@@ -1691,8 +1734,16 @@ void TensorExprKernel::runKernel(Stack& stack) {
 
   // Update the stack.
   drop(stack, nInputs_);
+
+  int64_t idx = 0;
   for (auto& o : outputs) {
-    push_one(stack, std::move(o));
+    if (isOutputScalar_[idx++]) {
+      // Scalar outputs are returned as 0-dim tensors, we need to extract the
+      // scalar value from them
+      push_one(stack, o.item());
```

Comment:
perf nit: `at::native::item`

Reply:
Such change caused some builds to fail, so I'm gonna go back to the original version for now. We can optimize performance separately.

Reply:
hmm, sad :/